src/aws/osml/models/centerpoint/app.py

```diff
@@ -4,21 +4,89 @@
 import os
 from json import dumps
 
+import numpy as np
 from flask import Flask, Response, request
+from matplotlib.backends.backend_agg import FigureCanvasAgg
+from matplotlib.figure import Figure
+from matplotlib.patches import CirclePolygon
 
-from aws.osml.models.server_utils import detect_to_geojson_dict, load_image, setup_server
+from aws.osml.models.server_utils import detect_to_geojson, load_image, setup_server
 
 app = Flask(__name__)
 
 # Optional ENV configurations
 BBOX_PERCENTAGE = float(os.environ.get("BBOX_PERCENTAGE", 0.1))
+ENABLE_SEGMENTATION = bool(os.environ.get("ENABLE_SEGMENTATION", False))
 
 
-def gen_center_point_detect(width: int, height: int, bbox_percentage: float) -> dict:
+def gen_center_bbox(width: int, height: int, bbox_percentage: float) -> list:
```

Comment:
`list` -- should be `List[int]` from `import typing`

Reply:
Will update